src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -550,23 +550,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
             End If
 
             Dim Here = 1
-            While CanGet(Here)
-                ch = Peek(Here)
-                If IsWhitespace(ch) Then
-                    Here += 1
-                Else
-                    Exit While
+            Dim atNewLine As Boolean
+            Dim FoundCommentAfterUnderscore As Boolean = False
+            ' Line continuation is valid at the end of the line, or at the end of the file, or followed by a trailing comment.
+            ' Eg.  LineContinuation ( EndOfLine | EndOfFile | LineContinuationComment)
+            If CanGet(Here + 1) AndAlso IsWhitespace(Peek(Here)) AndAlso IsSingleQuote(Peek(Here + 1)) Then
+                If CheckFeatureAvailability(Feature.CommentsAfterLineContinuation) Then
+                    tList.Add(MakeLineContinuationTrivia(GetText(1)))
+                    tList.Add(MakeWhiteSpaceTrivia(GetText(Here)))
+                    ScanCommentIfAny(tList)
+                    ch = Peek()
+                    atNewLine = IsNewLine(ch)
+                    FoundCommentAfterUnderscore = True
                 End If
-            End While
-
-            ' Line continuation is valid at the end of the
-            ' line or at the end of file only.
-            Dim atNewLine = IsNewLine(ch)
-            If Not atNewLine AndAlso CanGet(Here) Then
-                Return False
             End If
+            If Not FoundCommentAfterUnderscore Then
+                While CanGet(Here)
+                    ch = Peek(Here)
+                    If IsWhitespace(ch) Then
+                        Here += 1
+                    Else
+                        Exit While
+                    End If
+                End While
 
-            tList.Add(MakeLineContinuationTrivia(GetText(1)))
+                atNewLine = IsNewLine(ch)
+                If Not atNewLine AndAlso CanGet(Here) Then
+                    Return False
+                End If
+
+                tList.Add(MakeLineContinuationTrivia(GetText(1)))
```

Comment:
this looks overly restrictive.  what if someone wanted to write:

```vb
me.Foo() _                      ' comment
     .Bar(a, b, c, d, e, f) _   ' that extends to here
```

Reply:
They will get an error (hopefully one that explains that only one space is required). If the spec changes I could that the above to work for any whitespace including tabs.

Reply:
> They will get an error (hopefully one that explains that only one space is required).

Reply:
Note: looking at teh link to the existing discussion, it starts with this exact example:

![image](https://user-images.githubusercontent.com/4564579/42783774-d759efce-8901-11e8-8522-73d03ec6c146.png)

It looks like a user would not actually be able to write out hte examplar in the way the original issue wanted.  So this doesn't seem like the spec is actually fitting what is being asked for.

Reply:
In https://github.com/dotnet/vblang/issues/65, I took "a space" discussion literally so tab and multiple whitespace would be invalid.

![image](https://user-images.githubusercontent.com/12789369/42784106-f353d46e-8902-11e8-84b4-169227aa140e.png)

Reply:
Yeah.   i wouldn't take that literally.  it sounds like they were just saying what they thought looked best.

I mean, if you comment in C#, i would prefer a space, so taht you write ```foo // comment```.  But that doesn't mean i would disallow ```foo//comment```.

Reply:
I think VB requires at least 1 space because _ is allowed at the end of a variable name. I can look into allowing multiple spaces after the _ and before the single quote.

Reply:
@CyrusNajmabadi  I have updated to allow 0 or more spaces after I saw this comment.